src/js/mocks/constant.js

```diff
@@ -0,0 +1,4 @@
+export const VALID_USER = 'user';
```

Comment:
can this go under src/js/test/mocks instead?

Reply:
ok